### PR TITLE
Disable language selection

### PR DESCRIPTION
**Pull request by sunghwan-park-fw**
- preload translations
- remove language selection
- change app name to self-assessment tool

### CODE REVIEW

File: apps/web/public/locales/en/translation.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "Fever or chills": "Fever or chills",
   "Go back": "Go back",
   "Headache": "Headache",
-  "Header Title": "COVID-19 Symptom Checker",
+  "AppName": "COVID-19 Self-Assessment Tool",
```
Comment (hitankar-freshworks) [inline apps/web/public/locales/en/translation.json:12]: probably need consistency to whether do "AppName" or "App Name"

**Review by hitankar-freshworks — APPROVED** [review]
1 comment. we can address this later!

**Comment by hitankar-freshworks** [comment]
image
please let me know if you got to remove this as well

File: apps/web/public/locales/en/translation.json (hunk 1 above)
Comment (sunghwan-park-fw) [reply apps/web/public/locales/en/translation.json:12]: I'll do that in [HAPP-455](https://freshworks.atlassian.net/browse/HAPP-455)

**Comment by sunghwan-park-fw** [comment]
> 

Removed language selection from header and landing page.

**Pull request merged**